src/psyclone/tests/domain/nemo/transformations/nemo_arrayrange2loop_trans_test.py

```diff
@@ -43,16 +43,15 @@
 
 from psyclone.psyir.nodes import Assignment, CodeBlock, BinaryOperation, Call
```

Comment:
Please could you put these into alpha order.

Reply:
Done